WordPress/src/main/java/org/wordpress/android/ui/mlp/ModalLayoutPickerListDiffCallback.kt

```diff
@@ -9,10 +15,24 @@ class ModalLayoutPickerListDiffCallback(
     private val oldList: List<ModalLayoutPickerListItem>,
     private val newList: List<ModalLayoutPickerListItem>
 ) : Callback() {
+    object Payload
+
     override fun areItemsTheSame(oldItemPosition: Int, newItemPosition: Int): Boolean {
         val newItem = newList[newItemPosition]
         val oldItem = oldList[oldItemPosition]
-        return oldItem == newItem
+        if (newItem.type == oldItem.type) {
+            return when (newItem.type) {
+                TITLE -> (newItem as? Title)?.visible == (oldItem as? Title)?.visible
```

Comment:
I'm not sure I understand this condition for `title`. If one title is invisible and the other is visible that means the list item can't be recycled? Could you please clarify it a bit, thanks!

Reply:
You are right. This check is not needed. Fixed with baee77e672b474f115007ee6ae593172d4065660